src/components/ProjectCard/ProjectCard.styles.js

```diff
@@ -33,6 +33,7 @@ export const ProjectCardHeader = styled('div')`
   justify-content: space-between;
   background: ${theme.color.primaryColor};
   color: ${theme.color.white};
+  gap: 20px;
```

Comment:
Can you use one of the theme properties instead of a hard-coded value?
```suggestion
   gap: ${theme.spacing.large}
```